utils.py

```diff
@@ -103,6 +103,104 @@ def store_boe_pdfs(base_url, minutes_url):
     return
 
 
+def get_boe_minutes(base_url="https://comptroller.baltimorecity.gov", minutes_url_part="/boe/meetings/minutes", save_folder='BoE Minutes PDFs', start_dir=os.path.dirname(__file__)):
+    """
+    base_url:
+        url for the comptrollers website
+
+    minutes_url:
+        the extra bit of the URL that specifies the minutes "landing page"
+
+        https://comptroller.baltimorecity.gov/boe/meetings/minutes
+
+    save_folder:
+        the name of the folder used to hold the minutes PDFs, shouldn't have any slashes
+
+    start_dir:
+        defaults to the directory the calling script is in. used to make a full path (relative to the calling script or absolute) with save_folder
+    """
+    # print(start_dir)  # print the directory this script is in
+    os.chdir(start_dir)  # set the cwd to avoid any OS/python shenanigans
+    pdf_save_dir = f'{start_dir}/{save_folder}/'  # set save directory for pdfs
+    os.makedirs(pdf_save_dir, exist_ok=True)  # create directory if it doesn't exist (no error if it does)
+    minutes_url = base_url + minutes_url_part
+
+    # because whoever wrote the BoE website didn't make it easy to scrape
+    # date_regex = re.compile(r'([(January|February|March|April|May|June|July|August|September|October|November|December)]) (\d{1,2}), (\d{4})', re.IGNORECASE)
+    date_regex = re.compile(r'(\w*)\s+(\d{1,2})\D*(\d{4})', re.IGNORECASE)
```

Comment:
I can help with that. 
(\w*)   grabs any number of words (\w) because while there should only be one, it's possible there's some erroneous whitespace in the month
\s+   looks for at least one whitespace character, to identify the break between the month and the day
(\d{1,2})   finds one- or two-digit numbers (the day)
\D*   finds any non-digit character to separate the day from the year; during testing I found that looking for a space or a comma didn't always grab everything
(\d{4})   finds a four-digit number (the year)
case-insensitive searching to avoid getting snagged on typos (e.g.: "may" not matching "May")

Reply:
Yup, I added this is in my most recent commit!